.gitlab-ci.yml

```diff
@@ -120,6 +120,8 @@ test-prep:
     expire_in: 30 days
 
 test:staging-deployment:
+  # Allow failure until timing issues are resolved
+  allow_failure: true
```

Comment:
@mzedel @alfrunes Please let's create a JIRA ticket (and add a reference here in the comment) before merging the PR.